xmipp

```diff
@@ -767,7 +767,7 @@ def configCuda(configDict):
             else:
                 configDict["CXX_CUDA"] = candidate
             configDict["NVCC_CXXFLAGS"] = ("--x cu -D_FORCE_INLINES -Xcompiler -fPIC "
-                                           "-ccbin %(CXX_CUDA)s -std=c++11 "
+                                           "-ccbin %(CXX_CUDA)s -std=c++11 --expt-extended-lambda "
```

Comment:
seems tthat we don't need this flag, unless you use the both `__device__ __host__` lambda, see paragraph about the [news](https://devblogs.nvidia.com/new-compiler-features-cuda-8/) in Cuda 8

Reply:
If I remove the flag, then I get _"error: `__host__` or `__device__` annotation on lambda requires --expt-extended-lambda nvcc flag"_, and the lambda is  `__device__` lambda.

Reply:
confirmed